pepper-and-carrot-ep6.msrt

```diff
@@ -699,7 +699,7 @@
 [eng] Incredible!
 [epo] Nekredeble!
 [jbo] .i spaji
-[fra] Incroyable !
+[fra] Inoui !
```

Comment:
There should be an accent on the last 'i' here if I am not mistaken. I am skeptical however that this change is actually beneficial for the lipsync, since the original Russian word seems to resemble Incroyable more to me. As for the how all of these changes in this PR affect the meaning and flow, I don't speak French well enough to say.